src/app/qgisapp.cpp

```diff
@@ -1704,7 +1704,7 @@ void QgisApp::createActions()
   connect( mActionHideAllLayers, SIGNAL( triggered() ), this, SLOT( hideAllLayers() ) );
   connect( mActionShowSelectedLayers, SIGNAL( triggered() ), this, SLOT( showSelectedLayers() ) );
   connect( mActionHideSelectedLayers, SIGNAL( triggered() ), this, SLOT( hideSelectedLayers() ) );
-  connect( mActionHideDeselectedLayers, &QAction::triggered, this, &QgisApp::hideDeselectedLayers );
```

Comment:
Sorry, my mistake. For some reason, at a point, the Icon for hideDeselectedLayers was not showing up. I was trying to find a reason for it. Apparently, only saving the icon again made it work and then I forgot to revert this.

Thanks for the warning. It's fixed now.